src/restore.c

```diff
@@ -38,11 +38,11 @@
 static restore_config_t* g_conf;
 static restore_status_t* g_status;
 
-
 //==========================================================
 // Forward Declarations.
 //
 
+static restore_status_t* start_restore(restore_config_t *conf);
```

Comment:
Is it normal to have a forward declaration of your new restore_run() function?

Reply:
start_restore is now a local helper, it's pattern in these files to forward declare local helpers and is required if they aren't listed in any imported header files. I should add `static` to the definition though, will do.